### DISCUSSION TITLE
How to define a 'Point' in MOOSE?

### DISCUSSION THREAD

**Discussion by Minjiang-Zhu** [idaholab/moose#24815] [Q&A General] [answered]
Hi, I wonder how the "Point" class is defined in Moose. "Point" appears in ""FindValueOnLineMod.C".

**Comment by Minjiang-Zhu** [comment, marked as answer]
One may define a point by:
Point p;
p(0) = 0; p(1) = 0; p(2) = 0;

**Comment by GiudGiud** [comment]
Hello

Please have a look at the doxygen for these questions:
https://mooseframework.inl.gov/docs/doxygen/libmesh/classlibMesh_1_1Point.html

the constructors let you initialize the point as you create it

Guillaume
